pyuvdata/utils.py

```diff
@@ -1097,7 +3116,74 @@ def get_lst_for_time(jd_array, latitude, longitude, altitude):
                 "extrapolated value"
             )
             times.delta_ut1_utc = delta
-    lst_array = times.sidereal_time("apparent").radian[reverse_inds]
+    if astrometry_library == "erfa":
+        # This appears to be what astropy is using under the hood,
+        # so it _should_ be totally consistent.
+        gast_array = erfa.gst06a(times.ut1.jd, 0.0, times.tt.jd, 0.0)
+        lst_array = np.mod(gast_array + (longitude * (np.pi / 180.0)), 2.0 * np.pi)[
+            reverse_inds
+        ]
+    elif astrometry_library == "astropy":
+        lst_array = times.sidereal_time("apparent").radian[reverse_inds]
+    elif astrometry_library == "novas":
+        # Import the NOVAS library only if it's needed/available.
+        try:
+            from novas import compat as novas
+            from novas.compat import eph_manager
+            import novas_de405  # noqa
+        except ImportError as e:  # pragma: no cover
+            raise ImportError(
+                "novas and/or novas_de405 are not installed but is required for "
+                "NOVAS functionality"
+            ) from e
+
+        # We import this module just to make sure that it exists, and the check below
+        # forces flake8 not to freak out that we tried loading it in the first place
+
```

Comment:
I think this comment is out of date and if any of it is kept it should be moved before the import on line 3133.

Reply:
Whoops, that was left by mistake, removed in 2d8ae055.